src/openrct2-ui/windows/Sign.cpp

```diff
@@ -482,44 +505,48 @@ rct_window * window_sign_small_open(rct_windownumber number){
  *
  *  rct2: 0x6E6145
  */
-static void window_sign_small_mouseup(rct_window *w, rct_widgetindex widgetIndex)
+static void window_sign_small_mouseup(rct_window * w, rct_widgetindex widgetIndex)
 {
-    rct_banner* banner = &gBanners[w->number];
-    sint32 x = banner->x << 5;
-    sint32 y = banner->y << 5;
+    rct_banner * banner = &gBanners[w->number];
+    sint32       x      = banner->x << 5;
+    sint32       y      = banner->y << 5;
 
     rct_string_id string_id;
 
-    rct_tile_element* tile_element = map_get_first_element_at(x / 32, y / 32);
+    rct_tile_element * tile_element = map_get_first_element_at(x / 32, y / 32);
 
-    switch (widgetIndex) {
+    switch (widgetIndex)
+    {
     case WIDX_CLOSE:
         window_close(w);
         break;
     case WIDX_SIGN_DEMOLISH:
-        while (1){
-            if (tile_element_get_type(tile_element) == TILE_ELEMENT_TYPE_WALL) {
-                rct_scenery_entry* scenery_entry = get_wall_entry(tile_element->properties.wall.type);
-                if (scenery_entry->wall.scrolling_mode != 0xFF){
+        while (1)
+        {
+            if (tile_element_get_type(tile_element) == TILE_ELEMENT_TYPE_WALL)
+            {
+                rct_scenery_entry * scenery_entry = get_wall_entry(tile_element->properties.wall.type);
+                if (scenery_entry->wall.scrolling_mode != 0xFF)
+                {
                     if (tile_element->properties.wall.banner_index == w->number)
                         break;
                 }
             }
             tile_element++;
         }
         gGameCommandErrorTitle = STR_CANT_REMOVE_THIS;
-        game_do_command(
-            x,
-            1 | ((tile_element->type & 0x3) << 8),
-            y,
-            (tile_element->base_height << 8) | (tile_element->type & 0x3),
-            GAME_COMMAND_REMOVE_WALL,
-            0,
-            0);
+        game_do_command(x,
+                        1 | ((tile_element->type & 0x3) << 8),
```

Comment:
Not a fan of how this indentation style depends on the length of the function name.

Reply:
I think the style should change to be always next line, 4 in. Then if a function is renamed globally - the indent remains correct.

Reply:
I agree. @janisozaur can you change this?

Reply:
ok, I'll check if clang-format supports it

Reply:
If it doesn't then just turn it off altogether.